content/en/blog/2024/kubecon-china-2024.md

```diff
@@ -0,0 +1,53 @@
+---
+title:
+  OpenTelemetry Talks at Kubecon + CloudNativeCon + Open Source Summit + AI_dev
+  China 2024
+linkTitle: Kubecon China 2024
+date: 2024-06-25
+# prettier-ignore
+cSpell:ignore: Alhamdani Censhare EBPF Hrabusa Husni Huxing Jiahang Krom Sianturi Wanqi Zhang Zihao Ziyi
+author: '[Tiffany Hrabusa](https://github.com/tiffany76)'
+draft: true # TODO: remove this line once your post is ready to be published
+sig: Communications SIG
+---
+
+Join members of the OpenTelemetry community at
+[KubeCon + CloudNativeCon + Open Source Summit + AI_dev](https://events.linuxfoundation.org/kubecon-cloudnativecon-open-source-summit-ai-dev-china/)
+in Hong Kong from August 21 to 23, 2024.
+
+Read on to learn about all the OpenTelemetry events at KubeCon China 2024.
+
+This post will be updated as we receive notice of other activities. Check back
+before the conference begins!
+
+## Maintainer sessions
+
+- **[OpenTelemetry Community Update | OpenTelemetry社区更新](https://sched.co/1eYcJ)**<br>
+  by Zihao Rao & Huxing Zhang, Alibaba Cloud; Trask Stalnaker, Microsoft<br>
```

Comment:
Oops, I shouldn't be listed as a speaker, I've contacted CNCF to remove my name from the session (I was only coordinating / submitting it on Zihao and Huxing's behalf)

```suggestion
  by Zihao Rao & Huxing Zhang, Alibaba Cloud<br>
```